t3f/kronecker_test.py

```diff
@@ -1,30 +1,28 @@
 import numpy as np
 import tensorflow as tf
 
-from t3f import tensor_train
-from t3f import kronecker as kr
-from t3f import ops
-from t3f import initializers
-from t3f import variables
+import t3f
```

Comment:
Please, do not import t3f anywhere in the library. Again, not sure why it's bad, but noone does it)